charter.md

```diff
@@ -0,0 +1,32 @@
+# Style Team
+
+## Aims: Evolving the Rust Style over time
+
+1. Selecting styling for new Rust constructs
+2. Evolving the existing style over the course of Rust editions (without breaking backwards compatibility)
+3. Defining mechanisms to evolve the Rust style while taking backwards compatibility into account, such as via Rust editions or similar mechanisms
+
+## Domains
+
+1. [Style Guide]
+2. [T-style team repo](https://github.com/rust-lang/fmt-rfcs/)
+3. [#T-style] and #T-style/private zulip streams
+4. `rust-lang` issues with the `T-style` or `I-style-nominated` labels
+
+## Membership
+
+* Caleb Cartwright ([@calebcartwright](https://github.com/calebcartwright))
+* Jane Losare-Lusby ([@yaahc](https://github.com/yaahc))
+* Josh Triplett ([@joshtriplett](https://github.com/joshtriplett))
+* Michael Goulet ([@compiler-errors](https://github.com/compiler-errors))
```

Comment:
I think the list of current members belongs outside of the charter itself, and should be in the top-level README or similar.

Reply:
Or just a link to https://www.rust-lang.org/governance/teams/lang#Style%20team

Reply:
@compiler-errors That sounds even better. :)